include/core/grids.h

```diff
@@ -59,6 +58,27 @@ read_mesh_and_manifolds(
   const bool                                            &restart,
   const BoundaryConditions::BoundaryConditions          &boundary_conditions);
 
+/**
+ * @brief Completely set-up a mesh and its manifold for rotor-stator domains
+ *
+ * @param triangulation [stator] The triangulation to which a grid is attached. It will be substituted by the merged rotor-stator triangulation
```

Comment:
```suggestion
 * @param[in,out] triangulation [stator] The triangulation to which a grid is attached. It will be substituted by the merged rotor-stator triangulation
```